crates/proto/src/h3/h3_client_stream.rs

```diff
@@ -320,8 +320,9 @@ impl H3ClientStreamBuilder {
         name_server: SocketAddr,
         server_name: Arc<str>,
         path: Arc<str>,
+        disable_grease: bool,
```

Comment:
I think this should be moved to be a part of the builder struct, and set with a builder method.